### ISSUE TITLE
Better docs for server setup and server URL assignment

### ISSUE THREAD

**Issue by jamescostian** [exoframejs/exoframe#204]
I can run `exoframe` (to deploy) and it works. I get something like this on the server:

```
$ docker ps
CONTAINER ID        IMAGE                   COMMAND                  CREATED             STATUS              PORTS                                      NAMES
2127c785192f        exo-foobar-nec:latest   "npm start"              5 minutes ago       Up 5 minutes        80/tcp                                     exo-foobar-nec-96432ae0
6afee26a9edf        traefik:latest          "/traefik -c /dev/nu…"   About an hour ago   Up 7 minutes        0.0.0.0:80->80/tcp, 0.0.0.0:443->443/tcp   exoframe-traefik
a2e60194ccd0        exoframe/server         "/exoframe-server"       About an hour ago   Up 6 minutes        8080/tcp                                   exoframe-server
```

But after that, I can't run `exoframe -u`, or `exoframe ls`, or `exoframe update`, or even `exoframe login`. They all tell me that they ran into a 502 if my app hasn't finished starting up yet, or a 404 if it has. If I remove the container with my code in it via `docker rm -f 2127c785192f`, then the website stops showing my app (understandably) and I can `exoframe update`, or run any of the other commands! I can even deploy again, using `-u` or not using `-u`, but either way, I end up right at the same problem again - I can't deploy again or access exoframe at all!

`docker logs exoframe-traefik` gives me nothing. `docker logs exoframe-server` gives some output, but no errors. Here's its full log:

```
[exoframe-server] › ℹ  info                         Initializing docker services...
[exoframe-server] › ℹ  info                         Traefik instance started..
[exoframe-server] › ℹ  info                         Server running at: 8080
[exoframe-server] › ℹ  info                         Initializing docker services...
[exoframe-server] › ℹ  info                         Traefik already running, docker init done!
[exoframe-server] › ℹ  info                         Server running at: 8080
[exoframe-server] › ℹ  info                         Initializing docker services...
[exoframe-server] › ℹ  info                         Traefik already running, docker init done!
[exoframe-server] › ℹ  info                         Server running at: 8080
[exoframe-server] › ℹ  info                         Initializing docker services...
[exoframe-server] › ℹ  info                         Traefik already running, docker init done!
[exoframe-server] › ℹ  info                         Server running at: 8080
```

Also, I've noticed that every time I reboot, `exoframe-server` isn't running, and I have to `docker start exoframe-server`.

**Comment by jamescostian** [comment]
I also just tried [this sample node project](https://github.com/exoframejs/exoframe-server/tree/master/test/fixtures/node-project) and had the same problem.

**Comment by jamescostian** [comment]
Never mind, it's my fault. I noticed in the examples that `exoframe.your-host.com` was used, and so I used something like this:

```
docker run -d \
...
  --label traefik.frontend.rule=Host:your-host.com \
...
```

Because I didn't prefix it with `exoframe.`, traefik didn't route things like `/login` to exoframe, it just routed them to my application.

**Comment by jamescostian** [comment]
I think it would be great if the documentation could be a little more specific about this. For example, [the README](https://github.com/exoframejs/exoframe) says `exoframe endpoint http://you.server.url` which makes it seem like you plug in the URL you want as that endpoint, when in reality, you need to have a subdomain in there that's just for exoframe. The [hackernoon post](https://hackernoon.com/continuous-deployment-for-your-node-js-projects-in-10-minutes-with-exoframe-bdf48340c1be) also doesn't emphasize the importance of this, and the [Server Installation page](https://github.com/exoframejs/exoframe/blob/master/docs/ServerInstallation.md) tries to explain that label, but does not explain what happens if you don't start it with `exoframe.`

**Comment by yamalight** [comment]
Closing this now as it's been addressed in #208

**Comment by jamescostian** [comment]
Thanks! And I'd also like to say that I'm really enjoying exoframe and it's simplicity. Thank you so much for creating it!
